runtime/src/consensus/roothash.rs

```diff
@@ -121,6 +121,23 @@ impl MessageEvent {
     }
 }
 
+/// Information about how a particular round was executed by the consensus layer.
+#[derive(Clone, Debug, Default, PartialEq, Eq, Hash, Serialize, Deserialize)]
+pub struct RoundResults {
```

Comment:
should we add a simple serialization test case for the `RoundResults` struct to match go side, similarly as we have for other structs here?

Reply:
Good idea, added.